client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/unified-design-picker.tsx

```diff
@@ -291,6 +302,10 @@ const UnifiedDesignPickerStep: Step = ( { navigation, flow } ) => {
 					: undefined,
 		} );
 
+		const isEnabledStyleSelection =
```

Comment:
I think we can move this up so that the value can be reused by the `enabled` option for `useStarterDesignBySlug()` above, what do you think?